### PR TITLE
Np 121 fe parking flow arriving

### PR DESCRIPTION
**Pull request by maxkanin**
## **Ticket**

**JIRA ID:** [NP-121](https://parking-system.atlassian.net/browse/NP-121)

## Changes proposed in this pull request
- [IMPORTANT] Reviewer **will not review** any PR without summary of changes
- add Arriving UI
- Parking successful UI

please fill all the required fields from below Checklist otherwise reviewer **will not review** this PR
## Checklist
- [x] Screenshot evidence for both IOS and Android
- [x] Total change is less than 500 lines of code

## **Screenshot Evidence**

- Android
image
image

-IOS
image
-image

[NP-121]: https://parking-system.atlassian.net/browse/NP-121?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: screens/parkingFlow/Arriving.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+  return (
+    <View style={{ flex: 1 }}>
+      {region ? (
+        <>
+          <MapView
+            style={styles.map}
+            initialRegion={region}
+            provider={PROVIDER_GOOGLE}
+          ></MapView>
```
Comment (TaeCV) [inline screens/parkingFlow/Arriving.tsx:62]: Should we show the parking space location ?

Comment (maxkanin) [reply screens/parkingFlow/Arriving.tsx:62]: `const [region, setRegion] = useState<RegionType>();
  const { bookingRequest, parkingLot } = route.params;
  useLayoutEffect(() => {
    getLocation();
  }, []);

  const getLocation = async () => {
    const { lat, lng } = parkingLot.coord;
    setRegion({
      latitude: lat,
      longitude: lng,
      latitudeDelta: 0.0922,
      longitudeDelta: 0.0421,
    });
  };` initialRegion ใส่lat long จากparking lot แล้วมันไม่ได้ใช้แบบนี้ออ

Comment (TaeCV) [reply screens/parkingFlow/Arriving.tsx:62]: I misread it. You did it right

File: screens/parkingFlow/Arriving.tsx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+export type ArrivingProps = CompositeScreenProps<
+  NativeStackScreenProps<ParkingFlowStackParamList, "Arriving">,
+  NativeStackScreenProps<AuthenticatedStackParamList>
+>;
+const Arriving: React.FC<ArrivingProps> = ({ route }) => {
+  const [region, setRegion] = useState<RegionType>();
+  const { bookingRequest, parkingLot } = route.params;
+  useLayoutEffect(() => {
+    getLocation();
+  }, []);
```
Comment (TaeCV) [inline screens/parkingFlow/Arriving.tsx]: depends on parkingLot

File: screens/parkingFlow/Arriving.tsx [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+  const getLocation = async () => {
```
Comment (TaeCV) [inline screens/parkingFlow/Arriving.tsx]: do we need async here?

File: screens/parkingFlow/Arriving.tsx (hunk 2 above)
Comment (maxkanin) [reply screens/parkingFlow/Arriving.tsx]: done

File: screens/parkingFlow/Arriving.tsx (hunk 3 above)
Comment (maxkanin) [reply screens/parkingFlow/Arriving.tsx]: done

**Pull request merged**